app/blueprints/templates/routes.py

```diff
@@ -12,9 +18,32 @@
     template_folder="templates",
 )
 
+file_upload_path = os.path.join(os.path.dirname(__file__), "uplaoded_files")
+
+def json_import(file_path, template_name):
+    from app.import_config.load_form_json import load_json_from_file
 
-@template_bp.route("/view")
+    load_json_from_file(file_path=file_path, template_name=template_name)
+
+
+@template_bp.route("/view", methods=["GET", "POST"])
 def view_templates():
     sections = get_all_template_sections()
     forms = get_all_template_forms()
-    return render_template("view_templates.html", sections=sections, forms=forms)
+    form = TemplateUploadForm()
+    if form.validate_on_submit():
+        template_name = form.template_name.data
+        file = form.file.data
+        if not os.path.exists(file_upload_path):
+            os.makedirs(file_upload_path)
+        if file:
```

Comment:
Do we need to write the file to disk somewhere? Where will that be on AWS? Can't we just use the in-memory data from the upload and read the json? If we do need to write the file for some reason can we remove it after import please

Reply:
There are two options we can use
1) Save file in the container and deleter after the import /use flask-caching create a temp directory and then remove it.
2) If we need to refer to the file imported in future, then we may have to find a way to store in AWS.

Happy to change based on suggestions.

Reply:
As we discussed @NarenderRajuB  go ahead with the change & also check whether temp dir being deleted once done other wise these files will get piled up until a restart